hedera-node/configuration/preprod/api-permission.properties

```diff
@@ -44,10 +44,10 @@ systemUndelete=60
 freeze=58
 
 # Scheduling
-scheduleCreate=0-*                                                             
-scheduleDelete=0-*                                                             
-scheduleGetInfo=0-*                                                            
-scheduleSign=0-*   
+scheduleCreate=0-*
+scheduleDelete=0-*
+scheduleGetInfo=0-*
+scheduleSign=0-*
```

Comment:
Wow! Perhaps we should add a stripping code (left and right) when reading a properties file.